contactaccessrights/js/src/access-rights.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  var extPath = CRM.vars.contactAccessRights.baseURL + '/js/src/access-rights';
+  require.config({
+    urlArgs: 'bust=' + (new Date()).getTime(),
+    paths: {
+      'access-rights': extPath
+    }
+  });
+
+  require(['access-rights/app'], function (angular) {
+    var actions = angular.element('.crm-actions-ribbon ul#actions');
+    var app = angular.element('<li id="access-rights"><div id="bootstrap-theme"> \
+        <div ng-controller="AccessRightsCtrl as $ctrl"> \
+          <a href class="btn btn-secondary-outline text-uppercase pull-right" ng-click="$ctrl.openModal()"> \
```

Comment:
Well that's a good question. I didn't see any examples regarding the button, aside from the wireframe on which the "Manage roles and teams" button looks just like the modal buttons. I'll ask Natsuki about it